src/main/java/org/example/model/User.java

```diff
@@ -0,0 +1,92 @@
+package org.example.model;
+
+import org.apache.log4j.Logger;
+import org.example.model.Observer.DesiredPath;
+
+import java.util.Objects;
+
+public class User {
+    private static final Logger logger = Logger.getLogger("GLOBAL");
+    private Integer id;
+    private String name;
+    private String surname;
+    private String email;
+    private DesiredPath desiredPath;
+
+    public User() {
+    }
+
+    public User(Integer id, String name, String surname, String email) {
```

Comment:
Assuming User id is auto-incremental, we don't need id parameter in the constructor

Reply:
Done